### PR TITLE
Fix readme typos

### PR DESCRIPTION
**Pull request by xdissent**
Fixes some random typos in README I noticed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ This extension integrates [include-what-you-use](https://github.com/include-what
 
 ![IWYU](https://helly25.com/wp-content/uploads/2023/05/iwyu-animated.gif)
 
-2) Manually optimize the include files for the current C/C++ file by pressing `cmd-shit-P` + `I` + `W` + `Y` + `U` for command "`Include What You Use (current file)`". The extension will then lookup the current editor's file in the compile_commands.json file. If found it will then run the include-what-you-use tool. If that is successful it will then call the fix_includes.py script to apply the changes.
+2) Manually optimize the include files for the current C/C++ file by pressing `cmd-shift-P` + `I` + `W` + `Y` + `U` for command "`Include What You Use (current file)`". The extension will then lookup the current editor's file in the compile_commands.json file. If found it will then run the include-what-you-use tool. If that is successful it will then call the fix_includes.py script to apply the changes.
```
Comment (xdissent) [inline README.md:17]: I had a hard time fixing this one because I thought it was hilarious 😆

**Comment by helly25** [comment]
Thanks!

File: README.md (hunk 1 above)
Comment (helly25) [reply README.md:17]: :-)

**Review by helly25 — DISMISSED** [review]

**Comment by helly25** [comment]
Please follow the instructions on signing commits: https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Comment by xdissent** [comment]
Sorry about that! Just got a new MacBook Air this week and hadn't migrated my keys

**Review by helly25 — APPROVED** [review]

**Review by Fab-Cat — APPROVED** [review]

**Pull request merged**